src/test/rpc/AccountObjects_test.cpp

```diff
@@ -1032,13 +1032,134 @@ class AccountObjects_test : public beast::unit_test::suite
         BEAST_EXPECT(acct_objs_is_size(acct_objs(gw, jss::hashes), 0));
     }
 
+    void
+    testNFTsMarker()
+    {
+        testcase("NFTsMarker");
+
+        using namespace jtx;
+        Env env(*this);
+
+        Account const bob{"bob"};
+        env.fund(XRP(10000), bob);
+
+        unsigned nftsSize = 10;
+        for (unsigned i = 0; i < nftsSize; i++)
+        {
+            env(token::mint(bob, 0));
+        }
+
+        env.close();
+
+        auto createFakeMarker = [](AccountID const& issuer,
```

Comment:
might be useful to add some comments for what this lambda do and why its defined and where it can be used.